packages/dev-tools/src/styles/dev-tools.css

```
@@ -0,0 +1,602 @@
+	--c15t-dev-tools-border: var(--c15t-border, light-dark(#e8e8e8, #303030));
+	--c15t-dev-tools-text: var(--c15t-text, light-dark(#171717, #ededed));
+	--c15t-dev-tools-muted: var(--c15t-text-muted, light-dark(#737373, #a3a3a3));
+	--c15t-dev-tools-accent: var(--c15t-primary, light-dark(#335cff, #6685ff));
+	--c15t-dev-tools-danger: light-dark(#b42318, #fda29b);
+	--c15t-dev-tools-focus: var(--c15t-dev-tools-accent);
+	/* Inspection controls sit above c15t's banner and dialog layers. */
+	--c15t-dev-tools-z: calc(
+		max(
+				var(--consent-dialog-z-index, 999999999),
+				var(--iab-cd-z-index, 999999999)
+			) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)